src/app-layout/drawer/resizable-drawer.tsx

```diff
@@ -81,6 +81,7 @@ export const ResizableDrawer = ({
   return (
     <Drawer
       {...props}
+      id={activeDrawer?.id}
```

Comment:
Why was this not caught by any of our automated tests? I thought there are checks for invalid ARIA references. Is this particular component (or this part of the component) not covered?

Reply:
great question. I do not know in details in what pages do we run a11y checks, but this one has the following conditions: 
1. classic mode
2. an opened drawer (not tools panel)

Reply:
ok, yeah, I think we just don't have a page with one drawer opened by default. Let me add one